src/applications/vaos/appointment-list/components/ScheduleNewAppointmentRadioButtons.jsx

```diff
@@ -120,35 +123,41 @@ function ScheduleNewAppointmentRadioButtons({
       )}
 
       {!selectedOption && (
-        <span
-          aria-disabled="true"
-          className="vads-u-padding--0 va-action-link--disabled"
+        <button
+          aria-label="Choose an appointment type to start scheduling"
+          id="schedule-button"
+          type="submit"
```

Comment:
I think the `type` on both of these buttons should probably be `type="button"`, since there's no `form` element on this page to submit.

Reply:
Another good catch.